package.json

```diff
@@ -12,7 +12,7 @@
     "start": "npm run dev",
     "docs": "typedoc --options ./typedoc.js --module commonjs",
     "build": "rimraf dist && rollup -m -c --environment NODE_ENV:production && npm run test:es-check",
-    "build:stats": "npm run build -- --environment WITH_STATS:true && open stats.html",
+    "build:stats": "rimraf dist && rollup -m -c --environment NODE_ENV:production --environment WITH_STATS:true && npm run test:es-check && open bundle-stats/index.html",
```

Comment:
This script was failing as it tried passing the `--environment WITH_STATS: true` to `npm run test:es-check`.